README.org

```diff
@@ -62,6 +62,8 @@ Install the stylesheet of your choice according to your browser's method (e.g. u
   For [[http://planet.emacsen.org][Planet Emacsen]]
 - ~solarized-stackexchange-*.css~
   For StackExchange sites.  Not all sites are covered, but some of the main ones are.  Patches welcome.
+- ~solarized-sakai-dark-*.css~
```

Comment:
Sorry to be so picky, but the `-dark` part shouldn't be in there, because there are both dark and light CSS files generated.  :)

Reply:
aaah, 👍